src/panels/config/cloud/google-assistant/cloud-google-assistant.ts

```diff
@@ -122,23 +125,23 @@ class CloudGoogleAssistant extends LitElement {
                 .map((trait) => trait.substr(trait.lastIndexOf(".") + 1))
                 .join(", ")}
             </state-info>
-            <paper-toggle-button
+            <ha-switch
               .entityId=${entity.entity_id}
               .disabled=${!emptyFilter}
               .checked=${isExposed}
               @checked-changed=${this._exposeChanged}
```

Comment:
```suggestion
              @change=${this._exposeChanged}
```

And the handler needs updating